nncf/tensorflow/accuracy_aware_training/keras_model_utils.py

```diff
@@ -120,3 +120,5 @@ def configure_optimizers_fn():
                                                tensorboard_writer=tensorboard_writer,
                                                log_dir=log_dir)
     callbacks.on_train_end()
+
+    return acc_aware_training_loop
```

Comment:
I would recommend to return a 'History' object with accuracy aware statistics  or accuracy statistics alone to be consistent with the fit method.

Reply:
Made it return the final statistics object